lib/src/react16_suggestors/react_style_maps_updater.dart

```diff
@@ -29,144 +29,187 @@ import './react16_utilities.dart';
 class ReactStyleMapsUpdater extends GeneralizingAstVisitor
     with AstVisitingSuggestorMixin
     implements Suggestor {
+  static final _cssValueSuffixPattern =
+      new RegExp(r'\b(?:rem|em|ex|vh|vw|vmin|vmax|%|px|cm|mm|in|pt|pc|ch)$');
+
   @override
   visitCascadeExpression(CascadeExpression node) {
     super.visitCascadeExpression(node);
 
+    // UiProps instances can't be created via constructors, so we want to skip
+    // these to avoid false positives on classes with a style prop.
+    //
+    // Make sure to only skip if a `new`/`const` keyword is present, since
+    // component factory invocations can be parsed by Dart 2 as
+    // InstanceCreationExpression even though they're MethodInvocations in
+    // the resolved AST.
+    {
+      final target = node.target;
+      if (target is InstanceCreationExpression && target.keyword != null) {
+        return;
+      }
+    }
+
     for (Expression cascade in node.cascadeSections) {
-      if (!hasComment(cascade, sourceFile, willBeRemovedCommentSuffix)) {
```

Comment:
It's best to view these changes with [whitespace-agnostic diff](https://github.com/Workiva/over_react_codemod/pull/41/files?w=1).